llvm/lib/YkIR/CMakeLists.txt

```diff
@@ -6,4 +6,5 @@ add_llvm_component_library(LLVMYkIR
   YkPasses
   MC
   Support
+  CodeGen
```

Comment:
I didn't see this before. Why is this needed?

Reply:
actually its not needed anymore cause I don't have the cli arg here any more.

Reply:
missed that, sorry
updated :point_right: 340a9736b07fcb75b0b557c00f2433c55fba9a39